arcade/application.py

```diff
@@ -402,15 +402,17 @@ def rect(self) -> Rect:
         """Return a Rect describing the size of the window."""
         return LBWH(0, 0, self.width, self.height)
 
-    def run(self) -> None:
+    from typing import Optional
+
+    def run(self, view: Optional[View] = None) -> None:
```

Comment:
Use the modern `View | None`  notation here instead of `Optional` and remove the stray import.